examples/react-app/github/deploy.yml

```diff
@@ -0,0 +1,82 @@
+name: Architect Deployment
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    types:
+      - closed
+
+env:
+  COMPONENT_TAG: latest
+  KUBECONFIG_NAME: eks-kubeconfig
+  ARTIFACT_NAME: architect-artifact
+  POSTGRES_HOST_FILENAME: postgres-host.txt
+  KUBERNETES_CLUSTER_NAME: kubernetes-cluster-name.txt
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+
+jobs:
+  terraform_setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Set up terraform
+        uses: hashicorp/setup-terraform@v1.2.1
+        with:
+          terraform_version: 1.1.3
+          terraform_wrapper: false
+      - name: Initialize terraform
+        run: cd ./terraform && terraform init -backend-config="bucket=${{ secrets.REMOTE_STATE_BUCKET }}" -backend-config="region=${{ secrets.AWS_DEFAULT_REGION }}" -backend-config="key=${{ secrets.AWS_ACCESS_KEY_ID }}"
+      - name: Terraform plan
+        id: plan
+        run: cd ./terraform && terraform plan -no-color -out tfplan -var prefix=${{ secrets.PREFIX }} -var postgres_user=${{ secrets.POSTGRES_USER }} -var postgres_password=${{ secrets.POSTGRES_PASSWORD }} -var postgres_database=${{ secrets.POSTGRES_DATABASE }}
+      - name: Terraform apply
+        id: apply
+        run: cd ./terraform && terraform apply tfplan -no-color
+      - name: Set terraform outputs
+        id: set_terraform_outputs
+        run: |
+          cd ./terraform
+          POSTGRES_HOST=$(terraform output -json | jq -r .postgres_host.value)
+          echo $POSTGRES_HOST > ${{ env.POSTGRES_HOST_FILENAME }}
+          KUBERNETES_CLUSTER_NAME=$(terraform output -json | jq -r .kubernetes_cluster_name.value)
+          echo $KUBERNETES_CLUSTER_NAME > ${{ env.KUBERNETES_CLUSTER_NAME }}
+      - name: Upload postgres host artifact
+        uses: actions/upload-artifact@v2.3.1
+        with:
+          name: ${{ env.ARTIFACT_NAME }}
+          path: |
+            ./terraform/${{ env.POSTGRES_HOST_FILENAME }}
```

Comment:
Try using outputs instead of artifact? We use in our own github actions.
https://stackoverflow.com/questions/59175332/using-output-from-a-previous-job-in-a-new-one-in-a-github-action

Reply:
Related issue: https://github.community/t/environment-url-not-set-because-env-may-contain-secret/153708/17

I think I can change around a thing or two to make this go away, but of course someone could still hit this issue in the future